src/Avalonia.Controls/SystemDialog.cs

```diff
@@ -52,6 +52,11 @@ public class SaveFileDialog : FileDialog
         /// </summary>
         public string? DefaultExtension { get; set; }
 
+        /// <summary>
+        /// Gets or sets a value indicating whether to display a warning if the user specifies the name of a file that already exists.
+        /// </summary>
+        public bool? ShowOverwritePrompt { get; set; }
```

Comment:
I'm curious why is this nullable too? Doesn't seem to be required and isn't in WPF

Reply:
In the original issue, I noted that operating systems (or their respective shells) differ in default behavior if the property is unset. WPF is Windows only, so it doesn't need to care about the default behavior on any other OS.